### DISCUSSION TITLE
How to build ios cocoapods?

### DISCUSSION THREAD

**Discussion by Knoxvillekm** [yshrsmz/BuildKonfig#150] [Q&A]
I use same property for builds.
android - ./gradlew assembleRelease --project-prop buildkonfig.flavor=test
jvm - ./gradlew createDistributable --project-prop buildkonfig.flavor=test
This work fine.

ios - ./gradlew podInstall --project-prop buildkonfig.flavor=test
This not working.
If I change buildkonfig.flavor in gradle.properties its work too.

**Comment by mr-kew** [comment]
I have a similar issue. I used to use script build phase:
```
if [ $ENABLE_PREVIEWS = "YES" ]; then exit 0; fi
cd "$SRCROOT/.." && ./gradlew :shared:embedAndSignAppleFrameworkForXcode -P buildkonfig.flavor=prod
```

But since updating to kotlin to version 2.0.0 and BuildKonfig version 1.5.1, it just says:
```
'embedAndSign' task can not be used in a project with dependencies to pods.
```

I tried switching to any other task like `build`, `assemble` or `generateBuildKonfig`, it compiles & runs but the flavor is not set.

### Edit
I reverted back to the old kotlin version 1.19.20 and BuildKonfig 1.5.0 and any other task than `:shared:embedAndSignAppleFrameworkForXcode` also does not work. So I am probably gonna create [another thread for this](https://github.com/yshrsmz/BuildKonfig/discussions/151) as it is a different topic. 

@Knoxvillekm As for your issue I can only recomend to try using `embedAndSignAppleFrameworkForXcode` task instead of `podInstall`. It's just a blind guess tho.
